tests/integrations/raftstore/test_region_heartbeat.rs

```diff
@@ -173,6 +173,8 @@ fn test_region_heartbeat_timestamp() {
     assert!(reported_ts_now > reported_ts);
 }
 
+// FIXME(nrc) failing on CI only
```

Comment:
Any idea why it fails?

Reply:
> I have no idea! I can't reproduce locally, on an IDC machine, or on a K8s pod cloned from the CI where it fails (!) I intend to investigate further once this PR is landed and before turning on Prost.

Reply:
No idea! I can't reproduce locally, on an IDC machine, or on a K8s pod cloned from the CI where it fails (!) I intend to investigate further once this PR is landed and before turning on Prost.